pkg/api/validation/validation.go

```diff
@@ -1309,8 +1311,21 @@ func ValidatePodSecurityContext(securityContext *api.PodSecurityContext, spec *a
 
 	if securityContext != nil {
 		allErrs = append(allErrs, validateHostNetwork(securityContext.HostNetwork, spec.Containers, specPath.Child("containers"))...)
+		if securityContext.FSGroup != nil &&
+			!validation.IsValidGroupId(int32(*securityContext.FSGroup)) {
```

Comment:
what's worse - casting to int32 here or receiving int64?  I feel like receiving int64 is less ugly.